log/log.go

```diff
@@ -47,9 +47,10 @@ func (f *RawFormatter) Format(entry *logrus.Entry) ([]byte, error) {
 }
 
 func init() {
-	formatter := new(prefixed.TextFormatter)
+	formatter := new(logrus.TextFormatter)
```

Comment:
@titpetric Does this change the log output format? Specifically, if someone's parsing the log messages would they need to change their patterns?

Reply:
@excieve the change applies to values being correctly quoted (see changed _test.go file for example). Other than that the output remains unchanged and all existing patterns remain the same. We remove TTY output formatting only with DisableColors. This log format compatibility is also noted in the prefixed-logger project readme.

Reply:
There's an exception to the rule - if somebody tried to extract say, some json printf log, as it would now be quoted, they would have to adjust that parser accordingly. Within reason, this should be very rare.

Reply:
Thanks @titpetric! This makes perfect sense.